src/api/api.tsx

```diff
@@ -0,0 +1,81 @@
+import { UUID } from "crypto";
+
+export default class API {
+  // Singelton instance
+  static api: null | API;
+  api = null;
+
+  // Local Python backend
+  URL = "http://127.0.0.1:5000/skillsync/api";
+
+  //Resume
+  getResumesURL = () => `${this.URL}/resumes`;
+  getResumeURL = (id: UUID) => `${this.URL}/resumes/${id}`;
+  getResumeByPersonURL = (id: UUID) => `${this.URL}/resumes/person/${id}`;
+  addResumeURL = () => `${this.URL}/resumes`;
+  updateResumeURL = (id: UUID) => `${this.URL}/resumes/${id}`;
+  deleteResumeURL = (id: UUID) => `${this.URL}/resumes/${id}`;
+
+  //Cover Letter
```

Comment:
Please find a fitting name for the endpoint. Since we only need one endpoint for both files (see comment below)